### DISCUSSION TITLE
Welcome to TwoFactorAuth Discussions!

### DISCUSSION THREAD

**Discussion by RobThree** [RobThree/TwoFactorAuth#60] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by killrawr** [comment]
Is this package still in use? if so could I get a flow-chart of how it works? to better understand how to implement it?

**Reply by willpower232** [reply]
I suggest you have a look at https://github.com/RobThree/TwoFactorAuth/blob/master/demo/demo.php if you want the shortest description of how to make things happen. You can also have a look at the documentation website.
